src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -489,6 +489,8 @@ export default {
 		updateQuickFilters(filter) {
 			if (filter.gender !== undefined) {
```

Comment:
The default of `gender` is `null`, so we might want to just do a `if (filter.gender)` check here. Or does this also receive `null` when filters are cleared?

Reply:
Yeah, receive `null` and becomes a false.